### PR TITLE
Including FlytOS in Devguide

### PR DESCRIPTION
**Pull request by peaboard**
Request to include FlytOS under companion computers of the Devguide.
For reference please visit [flytbase](https://flytbase.com/).

### CODE REVIEW

File: en/companion_computer/FlytOS.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+custom data plumbing between onboard and offboard apps. FlytOS aims to provide a standard language for the drone application
+developers to talk to their drones.
+
+{% youtube %}https://www.youtube.com/watch?v=CZFVWDN5Gcc{% endyoutube %}
+
+
+![](../../assets/FlytOSArch.png)
+   
+
+Developer Tools
```
Comment (hamishwillee) [inline en/companion_computer/FlytOS.md]: Suggest you don't mix underlining and # for titles. Just use ## for level 2 titles, ### for level 3 etc.

Comment (peaboard) [reply en/companion_computer/FlytOS.md]: Hi, thanks for the review. Have made the changes, could you please have a look.

**Comment by LorenzMeier** [comment]
This needs to be rebased. And we need to have some form of clarification that this is actually working with PX4 and with which release - and will work with the next. Who is responsible for outreach to Dronecode within your organisation?

**Comment by sharvashish** [comment]
Hi @LorenzMeier,

Yes, we will rebase and send the PR again. We have tested FlytOS with the latest stable release of PX4 namely v1.6-5 and we would make sure this works with latest stable releases of PX4 as and when they are released. I am responsible for making sure FlytOS is compatible with PX4, and would love to answer any of your further queries.

**Comment by peaboard** [comment]
Due to problems in this PR have made a new one please refer to [this](https://github.com/PX4/Devguide/pull/270). 
Thanks.

**Pull request closed without merging**